linera-client/src/unit_tests/chain_listener.rs

```diff
@@ -104,7 +104,7 @@ async fn test_chain_listener() -> anyhow::Result<()> {
     let config = ChainListenerConfig::default();
     let storage_builder = MemoryStorageBuilder::default();
     let clock = storage_builder.clock().clone();
-    let mut builder = TestBuilder::new(storage_builder, 4, 1, &mut signer).await?;
+    let mut builder = TestBuilder::new(storage_builder, 4, 1, signer.clone()).await?;
```

Comment:
Ah, cool. I think I added the `Arc<RwLock<Inner>>` to the `InMemorySigner` and forgot to remove this unnecessay `&mut` argument passing.